doc/scss/custom.scss

```diff
@@ -157,6 +157,17 @@ details.sd-dropdown {
   }
 }
 
+/* Download/laucher links and top hint (sphinx-gallery) */
```

Comment:
Maybe add a bit more details here? I guess this hides the note at the top (something like "Go at the bottom of the example to download things") and the download links at the bottom of the example. We don't need them because pydata-sphinx-theme puts the links on the right-hand-side panel.

Reply:
I've added links to the sphinx-gallery docs which I think has better explanations.